app/assets/images/logo.svg

```diff
@@ -0,0 +1 @@
+<svg height="2500" width="2500" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 512 512"><g fill="none" fill-rule="evenodd"><rect fill="#fabc05" height="512" rx="42.667" width="512"/><path d="M58.667 293.333L0 209.125V42.663C0 19.1 19.137 0 42.663 0h426.674C492.9 0 512 19.137 512 42.663l-282.667.004S58.667 305.14 58.667 293.333z" fill="#eb4335"/><path d="M303.3 340.528L199.876 512H42.633C19.087 512 0 492.94 0 469.328V209.125l54.537 74.854c28.457 48.33 81.024 80.762 141.167 80.762 40.548 0 77.822-14.86 106.533-39.29l5.348 5.349c-.918 3.235-2.337 6.5-4.284 9.728z" fill="#34a854"/><g fill-rule="nonzero"><path d="M194.667 85.333c-66.667 0-120 52.148-120 117.334C74.667 267.852 128 320 194.667 320c66.666 0 120-52.148 120-117.333 0-65.186-53.334-117.334-120-117.334z" fill="#4285f5"/><path d="M327.59 300.924l28.788 28.787 13.204-14.159 115.751 115.751-59.363 59.364-115.751-115.751 13.553-12.64-28.85-28.85c-27.526 20.496-61.677 32.712-98.52 32.712C105.605 366.138 32 292.532 32 201.735c0-90.796 73.605-164.402 164.402-164.402s164.402 73.606 164.402 164.402c0 37.142-12.414 71.549-33.213 99.19zM196.403 87.92c-63.231 0-113.817 50.585-113.817 113.816 0 63.232 50.586 113.817 113.817 113.817S310.22 264.967 310.22 201.735c0-63.231-50.585-113.816-113.817-113.816z" fill="#fff"/></g></g></svg>
```

Comment:
How about keeping it under `logo` folder like we did on Mia project? I think it will help if we add many different types of images later then we can find/understand them easily by looking at the parent folder.
![Screen Shot 2021-06-18 at 6 10 44 PM](https://user-images.githubusercontent.com/6076762/122552695-d0981e00-d060-11eb-98f9-d1c61b7a9bbc.png)

Reply:
It's a good remark! :) 
Though I'm always reluctant to solve problems that do not exists... yet. 

In the same way, I could create a v1 folder, in case one day we want to redesign the application identity/UI, but I believe we should do that **only** when there is a real problem to solve. 

In this project, I might add a favicon and if I go far crazy like Ankit, maybe a header-image. 
So I will have 3x images. 
--> Having 3 sub folders, each containing 1 single file, only bring complexity, not clarity. 

When I will start to have images that fit a similar category (ex: several logos to better match responsiveness, or dark/light UI), then it will be the right time to group these logos in a subdirectory. 

What do you think @junan ? :)

Reply:
I am fine with it. Then lets go with `YAGNI` and keep it as it is